### PR TITLE
Moench dacs defaults

### PR DESCRIPTION
**Pull request by thattil**
- changed dac names and default values to old moench values
- remove interface clk polarity at start up
- default speed is half speed, default values for adc offset and adc phase for different speeds (only half speed confirmed), adc vref voltage to 2.0 like G1
- connected adc pipeline to client
- moench: receiver- default frames per file is 100k and discard partial frames as default

### CODE REVIEW

File: slsReceiverSoftware/src/Implementation.cpp [hunk 1]
```diff
@@ -175,6 +175,11 @@ void Implementation::setDetectorType(const detectorType d) {
 
     SetThreadPriorities();
 
+    // special detector specific initializations
+    if (d == MOENCH) {
```
[the remark this answers is not in the document]
Comment (thattil) [reply slsReceiverSoftware/src/Implementation.cpp:179]: The default is no discard, but Anna wants moench to discard partial by default for moench. Thats why I put it here. The other option is to take it out of Implementation and bring it into GeneralData and have defaults for every detector. Shall I do that instead?

File: slsDetectorGui/src/qTabDeveloper.cpp [hunk 2]
```diff
@@ -184,6 +183,31 @@ void qTabDeveloper::SetupWidgetWindow() {
                 this, det, false,
                 "Temperature FPGA: ", getSLSIndex(detType, tempid++)));
             break;
+        case slsDetectorDefs::MOENCH:
+            dacWidgets.push_back(
+                new qDacWidget(this, det, true,
+                               "vbp_colbuf: ", getSLSIndex(detType, tempid++)));
+            dacWidgets.push_back(new qDacWidget(
```
[the remark this answers is not in the document]
Comment (thattil) [reply slsDetectorGui/src/qTabDeveloper.cpp:190]: Yea will try to do that!

[the remark this answers is not in the document]
Comment (thattil) [reply slsDetectorGui/src/qTabDeveloper.cpp]: done

File: slsReceiverSoftware/src/Implementation.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (thattil) [reply slsReceiverSoftware/src/Implementation.cpp]: done

**Comment by thattil** [comment]
Also added a full speed adc phase change from 0 to 300 degrees  (according to Anna's latest change in the corresponding issue)

File: slsDetectorGui/include/qTabDeveloper.h [hunk 3]
```diff
@@ -20,16 +20,49 @@ class qTabDeveloper : public QWidget, private Ui::TabDeveloperObject {
...
     void Initialization();
     void PopulateDetectors();
-    slsDetectorDefs::dacIndex getSLSIndex(slsDetectorDefs::detectorType detType,
-                                          int index);
 
     Detector *det;
     std::vector<qDacWidget *> dacWidgets;
     std::vector<qDacWidget *> adcWidgets;
+
+    const std::vector<std::string> eiger_dacs = {
```
Comment (erikfrojdh) [inline slsDetectorGui/include/qTabDeveloper.h:35]: Is this needed or can we get it from the API? Seems redundant but I might be missing something?

**Review by erikfrojdh — CHANGES_REQUESTED** [review]

File: slsDetectorGui/include/qTabDeveloper.h (hunk 3 above)
Comment (thattil) [reply slsDetectorGui/include/qTabDeveloper.h:35]: Ah you are totally right!

Comment (thattil) [reply slsDetectorGui/include/qTabDeveloper.h:35]: we dont have an adc list thought.. but the dacs atleast i should be able to get from the api

**Review by erikfrojdh — APPROVED** [review]

**Pull request merged**